src/developer-guide/nodejs-api.md

```diff
@@ -917,14 +916,14 @@ ruleTester.run("my-rule", myRule, {
 
 ---
 
-[configuration object]: ../user-guide/configuring/
-[builtin-formatters]: https://eslint.org/docs/user-guide/formatters/
+[configuration object]: ../user-guide/configuring/index
+[builtin-formatters]: ../user-guide/formatters/
 [third-party-formatters]: https://www.npmjs.com/search?q=eslintformatter
-[eslint-lintfiles]: #-eslintlintfilespatterns
-[eslint-linttext]: #-eslintlinttextcode-options
-[eslint-loadformatter]: #-eslintloadformatternameorpath
-[lintresult]: #-lintresult-type
-[lintmessage]: #-lintmessage-type
-[suppressedlintmessage]: #-suppressedlintmessage-type
-[editinfo]: #-editinfo-type
-[loadedformatter]: #-loadedformatter-type
+[eslint-lintfiles]: #◆-eslintlintfilespatterns
+[eslint-linttext]: #◆-eslintlinttextcode-options
+[eslint-loadformatter]: #◆-eslintloadformatternameorpath
+[lintresult]: #◆-lintresult-类型
+[lintmessage]: #◆-lintmessage-类型
+[suppressedlintmessage]: #◆-suppressedlintmessage-类型
+[editinfo]: #◆-editinfo-类型
+[loadedformatter]: #◆-loadedformatter-类型
```

Comment:
Why this change? It doesn't seem to work with `◆`.

Reply:
VSCode's Markdown Fragment Links check reminds me to do this, but it seems to be an error and I'll roll it back.

Reply:
I just found out that the anchor generated by eleventy does not support Chinese, which means that all the Chinese anchors used before are invalid. I may need to figure out how to fix it, but it won't be in this PR